### ISSUE TITLE
change name of profile to the file name

### ISSUE THREAD

**Issue by hiddify** [zzzgydi/clash-verge#343]
Currently, when we add a new profile, it will name it as Remote File. However it would be great if you name it as the file name for example all.yml or support the name query parameter in the url.

**Comment by zzzgydi** [comment]
You can rename the profile manually

**Comment by hiddify** [comment]
Yes, but we want to propose to use clash verge for users that have very limited knowledges. So it will help them to better distinguish different profiles.
